doc/dao/Getting Started with Development Environment & Running Primero.md

```diff
@@ -0,0 +1,291 @@
+# Development Environment
+
+> [!NOTE]
+> If you are already an experienced Rails developer and are familiar with Docker, grab a copy of the code from [github](https://github.com/primeroIMS/primero) and skip to the section [Building the Containers](#Building%20the%20Containers).
+
+In order to contribute to Primero, you need a working development environment. It is recommended to use a Mac or Linux operating system. If you are developing on Windows, you should consider using WSL or a VM.
+
+The instructions in this guide assume that you are running Ubuntu Linux. Primero supports both x86_64 and arm64 architectures, so if you're using an M1 Mac or another arm64 machine like a Raspberry Pi, you don't need to do anything special.
+
+## Cloning the Primero Repository
+
+To get a fresh copy of the code, you need to have [git](http://git-scm.com/book/en/v2/Git-Internals-Packfiles) installed.
+
+```bash
+sudo apt update
+sudo apt install git
+```
+
+Navigate to a directory in your shell where you would like to download Primero, and clone the repository.
+
+```bash
+git clone https://github.com/primeroIMS/primero.git
+```
+
+## Opening Primero in Visual Studio Code
```

Comment:
I definitely like this section. However we should warn folks about the Shopify Ruby extension - it's really raw!

Reply:
I've changed the link to point directly to the ruby-lsp extension rather than the pack as it's less heavy and opinionated. I've also written a warning about the extension broadly. Were there any particular sharp edges you think we should call out?

I also changed the launch.json to rely only on the ruby-lsp extension.